esmvalcore/cmor/_fixes/cmip6/cesm2_waccm.py

```diff
@@ -0,0 +1,23 @@
+"""Fixes for CESM2-WACCM model."""
+from ..fix import Fix
```

Comment:
This file is identical to the cesm2 module above. You can avoid the code duplication by replacing the code here with something like:
```python
from .cesm2 import Tas as BaseTas

class Tas(BaseTas)
    """Fixes for tas."""
```

It would also be good to avoid the same code duplication in the tests.

Reply:
Done, but not for the tests.